Assets/Scripts/Weapons/Arrow.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace Assets.Scripts.Weapons
+{
+    public class Arrow : MonoBehaviour
+    {
+        private float timeToLive = 1.0f;
+        private int damage = 10;
+        private List<int> hitableLayers = new List<int> { 8, 10 }; //ground, enemy
```

Comment:
We should not reference Layers by their numbers. Use LayerMask.GetMask("someName") to get the underlying int value. Hardcoding the int value is prone to errors due to layer order changes or layer additions / deletions.